tests/unit/orchestrator/test_strategies.py

```
@@ -0,0 +1,339 @@
+        strategy = FixedTrialsStrategy(num_trials=3)
+        assert strategy.should_continue([]) is True
+
+    def test_should_continue_with_partial_results(self):
+        """Test should_continue returns True when results < num_trials."""
+        strategy = FixedTrialsStrategy(num_trials=3)
+        results = [
+            RunResult(
+                label="run_0001",
+                success=True,
+                summary_metrics={"ttft_avg": 100.0},
+                artifacts_path=Path("/tmp/run_0001")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/run_0001"

(S108)